connect/resources/fulfillment_automation.py

```diff
@@ -73,7 +73,7 @@ def dispatch(self, request):
             return self.inquire(request.id)
 
         except FailRequest as fail:
-            return self.fail(request.id, reason=fail.message)
+            return self.fail(request.id, reason=str(fail))
```

Comment:
will break on python2.x if string will be with unicode

Reply:
Thanks, fixed. Since all Connect's exceptions derive from Message class, I have modified the class initializer so on Python 2, if a unicode string is passed, it will be stored as a UTF-8 str.